src/neo/Cryptography/Helper.cs

```diff
@@ -77,9 +77,9 @@ public static byte[] Base58CheckDecode(this string input)
             byte[] buffer = Base58.Decode(input);
             if (buffer.Length < 4) throw new FormatException();
             byte[] checksum = buffer.Sha256(0, buffer.Length - 4).Sha256();
-            if (!Neo.Helper.MemoryEquals(buffer.Skip(buffer.Length - 4), checksum.Take(4)))
+            if (!buffer.AsSpan(^4).SequenceEqual(checksum.AsSpan(..4)))
```

Comment:
SequenceEqual is very slow

Reply:
It is `MemoryExtensions.SequenceEqual()`. It is very fast.